DOCUMENTATION.md

```diff
@@ -0,0 +1,121 @@
+Fallback to X11 windowing system | `--socket=fallback-x11` and `--nosocket=fallback-x11` | Toggle | Permit (`--socket=fallback-x11`) or prohibit (`--nosocket=fallback-x11`) access to the application to show windows using X11 if Wayland is not available. **This overrides `--socket=x11` when used.**
+PulseAudio sound server | `--socket=pulseaudio` and `--nosocket=pulseaudio` | Toggle | Permit (`--socket=pulseaudio`) or prohibit (`--nosocket=pulseaudio`) the application to play sounds that use PulseAudio.
+D-Bus session bus | `--socket=session-dbus` and `--nosocket=session-dbus` | Toggle | Permit access to the application to the entire session bus (`--socket=session-dbus`); prohibit access to the application to the entire session bus (`--nosocket=session-dbus`).
+D-Bus system bus | `--socket=system-dbus` and `--nosocket=system-dbus` | Toggle | Permit (`--socket=session-dbus`) or prohibit (`--nosocket=session-dbus`) access to the application to the entire system bus.
+Secure Shell agent | `--socket=ssh-auth` and `--nosocket=ssh-auth` | Toggle | Permit (`--socket=ssh-auth`) or prohibit (`--nosocket=ssh-auth`) access to the application to SSH authentications.
+Smart cards | `--socket=pcsc` and `--nosocket=pcsc` | Toggle | Permit (`--socket=pcsc`) or prohibit (`--nosocket=pcsc`) access to the application to smart cards.
+Printing system | `--socket=cups` and `--nosocket=cups` | Toggle | Permit (`--socket=cups`) or prohibit (`--nosocket=cups`) access to the application to printing systems.
+
+#### Device
+
+List of devices available in the sandbox.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+GPU acceleration | `--device=dri` and `--nodevice=dri` | Toggle | Permit (`--device=dri`) or prohibit (`--nodevice=dri`) access to the application to graphics direct rendering located at `/dev/dri`.
+Virtualization | `--device=kvm` and `--nodevice=kvm` | Toggle | Permit (`--device=kvm`) or prohibit (`--nodevice=kvm`) access to the application to virtualization.
+Shared memory | `--device=shm` and `--nodevice=shm` | Toggle | Permit (`--device=shm`) or prohibit (`--nodevice=shm`) access to the application to shared memory.
+All devices | `--device=all` and `--nodevice=all` | Toggle | Permit (`--device=all`) or prohibit (`--nodevice=all`) access to the application to all devices.
+
+#### Allow
+
+List of features available to the application.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+Development syscalls | `--allow=devel` and `--disallow=devel` | Toggle | [NEEDS DESCRIPTION]
+Programs from other architectures | `--allow=multiarch` and `--disallow=multiarch` | Toggle | [NEEDS DESCRIPTION]
+Bluetooth | `--allow=bluetooth` and `--disallow=bluetooth` | Toggle | Permit (`--allow=bluetooth`) or prohibit (`--disallow=bluetooth`) access to the application to use bluetooth.
+Controller Area Network bus | `--allow=canbus` and `--disallow=canbus` | Toggle | [NEEDS DESCRIPTION]
```

Comment:
A good start could be to re-use `flatpak-override` docs for each of those overrides. See https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-override

e.g. _The multiarch feature allows the application to execute programs compiled for an ABI other than the one supported natively by the system. For example, for the x86_64 architecture, 32-bit x86 binaries will be allowed as well._

Reply:
That one is only for `Programs from other architectures` right?

Reply:
Correct!

Reply:
https://github.com/tchx84/Flatseal/pull/136/commits/1a3205204d9ee6da151386cb61ac8da2dcaa2fd8

Reply:
Devel: https://github.com/tchx84/Flatseal/pull/136/commits/b4cf67d39f5986a60a82bd6d0194e7a782b00bf6

Reply:
Canbus: https://github.com/tchx84/Flatseal/pull/136/commits/88e3cfd0a2893544c0651694ed645e3a5a239178

Reply:
This one is done right? Feel free to resolve it if it is.